org.kde.CrowTranslate.yaml

```diff
@@ -30,12 +30,13 @@ modules:
   - name: crow-translate
     buildsystem: cmake-ninja
     sources:
-      - type: git
-        url: https://invent.kde.org/office/crow-translate.git
-        commit: 2d5c588fa07576088c9a3aa533b49179523553d5
+      - type: archive
+        url: https://download.kde.org/stable/crow-translate/3.1.0/crow-translate-v3.1.0.tar.gz
+        sha256: 8cafa4a812079020839beaf0b133e08be6e47d6ffa1cf90f3417efdc7589c7a1
```

Comment:
I would like to keep the git source here. It's just very convenient that I receive an automatic update from Flathub once I draft a new tag.